status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.117.1a",
-    "commit-sha1": "b4bdfd3df6cf5fb91ab2d0e9f3b38e8d1b9703e5",
-    "src-sha256": "1s50q53p1j5ysnbj88c52zb2rh1ms21pnf6bjc7wja8gzcpih2wl"
+    "version": "6abbe98cd20b639d0c7298091f00262e876f3e17",
```

Comment:
revert? or necessary?

Reply:
It seems like we should revert.

I thought we should never merge pointing to a revision. Revisions change after push forces and are only relatively stable if pointing to a revision in the `develop` branch.

There's a particular case where the backend and frontend need to change in lockstep, but it doesn't seem to be the case here. As Andrea once explained to me, if the frontend requires a specific implementation of status-go that has not been released, then it should go more or less like the steps below:

1. Contributor opens the frontend PR when the backend already has the non-released code, and the contributor points to a specific backend revision (not version).
2. Contributor gets the frontend PR approved and potentially tested by QAs.
3. Contributor is now positive the backend implementation works, so they open a PR in the backend, get it approved and release a new version.
4. Contributor changes the frontend PR to point to the newly released backend version (not revision).
5. Contributor rebases the frontend PR, squash and merge, potentially requesting another round of review from QAs before merge.

Reply:
The backend impl. on status-go is already merged. I forgot to update it.